### PR TITLE
OCPQE-14562 fix cloud-cluster-profile

### PR DESCRIPTION
**Pull request by MayXuQQ**
fix cloud-cluster-profile: azure-qe
https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-mag-ipi-fullyprivate-p3-f28-long-duration-cloud/1640480333981814784

### CODE REVIEW

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 1]
```diff
@@ -5283,13 +5283,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-control-plane-p1-f4
-  reporter_config:
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:5286]: I think those removal are unexpected. Would you mind to double confirm ?

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 2]
```diff
@@ -20400,19 +20172,13 @@ periodics:
...
-    ci-operator.openshift.io/cloud: azure4
-    ci-operator.openshift.io/cloud-cluster-profile: azure-qe
+    ci-operator.openshift.io/cloud: azuremag
+    ci-operator.openshift.io/cloud-cluster-profile: azuremag-qe
     ci-operator.openshift.io/variant: amd64-nightly
     ci.openshift.io/generator: prowgen
     job-release: "4.13"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-mag-ipi-fullyprivate-p3-f28-long-duration-cloud
-  reporter_config:
```
Comment (jianlinliu) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:20410]: @sunzhaohua2 the setting is still needed, right?

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml (hunk 1 above)
Comment (MayXuQQ) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:5286]: `make jobs ` need more options? these lines were removed after `make jobs `. @liangxia

**Pull request merged**